cmake/std/atdm/ats1/environment.sh

```diff
@@ -83,50 +65,16 @@ export ATDM_CONFIG_SBATCH_EXTRA_ARGS="$ATDM_CONFIG_SBATCH_EXTRA_ARGS --exclude=n
 export OMP_NUM_THREADS=2
 
 # Common modules and paths
-module load cmake/3.14.6
-module load git/2.19.1
 export PATH=/projects/netpub/atdm/ninja-1.8.2/bin:$PATH
-module unload cray-mpich
-module load cray-mpich/7.7.6
-module unload cray-libsci
-module unload gcc
-
-if [[ "$ATDM_CONFIG_COMPILER" == "INTEL-19.0.4"* ]]; then
-  module load gcc/8.2.0
-  module unload intel
-  module load intel/19.0.4
-  #sparc_tpl_ext=${system_name}-${node_arch}_intel-19.0.4
-  #sparc_tpl_mpi_ext=${system_name}-${node_arch}_intel-19.0.4_mpich-7.7.6
-elif [[ "$ATDM_CONFIG_COMPILER" == "INTEL-18.0.5"* ]]; then
-  module unload intel
-  module load intel/18.0.5
-  #sparc_tpl_ext=${system_name}-${node_arch}_intel-18.0.5
-  #sparc_tpl_mpi_ext=${system_name}-${node_arch}_intel-18.0.5_mpich-7.7.6
-else
-  echo
-  echo "***"
-  echo "*** ERROR: COMPILER=$ATDM_CONFIG_COMPILER and KOKKOS_ARCH=$ATDM_CONFIG_KOKKOS_ARCH is not"
-  echo "*** a supported combination on this system!"
-  echo "*** Combinations that are supported: "
-  echo "*** > intel-19.0.4_mpich-7.7.6_openmp-HSW"
-  echo "*** > intel-19.0.4_mpich-7.7.6_openmp-KNL"
-  echo "*** > intel-18.0.5_mpich-7.7.6_openmp-HSW"
-  echo "*** > intel-18.0.5_mpich-7.7.6_openmp-KNL"
-  echo "***"
-  return
-fi
 
-export PATH=/usr/projects/hpcsoft/cle6.0/common/intel-clusterstudio/2019.4-068/compilers_and_libraries_2019/linux/mkl/bin:$PATH
-export LD_LIBRARY_PATH=/usr/projects/hpcsoft/cle6.0/common/intel-clusterstudio/2019.4-068/compilers_and_libraries_2019/linux/mkl/lib/intel64:$LD_LIBRARY_PATH
 # This linker path is needed for cmake's C, CXX, and Ftn compiler tests. This linker path is not shown
 # anywhere in SPARC's environment but is needed in ATDM's environment. Cmake's compiler tests are
 # ignoring LD_{CRAY,CRAYPAT,}_LIBRARY_PATH for some reason, so we add this path via LDFLAGS via
 # s.t. it is added to CMAKE_EXE_LINKER_FLAGS before trilinos's cmake config invokes PROJECT().
-export LDFLAGS="-L/opt/gcc/8.2.0/snos/lib/gcc/x86_64-suse-linux/8.2.0/ $LDFLAGS"
+export LDFLAGS="-L/opt/gcc/8.3.0/snos/lib/gcc/x86_64-suse-linux/8.3.0/ -lpthread $LDFLAGS"
 
-export CBLAS_ROOT=/usr/projects/hpcsoft/cle6.0/common/intel-clusterstudio/2019.4-068/compilers_and_libraries_2019/linux
-export COMPILER_ROOT=/usr/projects/hpcsoft/cle6.0/common/intel-clusterstudio/2019.4-068/compilers_and_libraries_2019/linux
-export MPI_ROOT=/opt/cray/pe/mpt/7.7.6/gni/mpich-intel/16.0
+# 2021-01-09 -- Resolve linker errors in Seacas and Percept
+export LDFLAGS="-L${MPI_ROOT}/lib -lmpich -lrt ${ATP_INSTALL_DIR}/lib/libAtpSigHandler.a ${ATP_INSTALL_DIR}/lib/libbreakpad_client_nostdlib.a $LDFLAGS"
```

Comment:
Wow, can't believe all of this is needed on the link line.  How is it that SPARC does not need this when it links?  Does the native SPARC Trilinos configuration not need this?  (But I guess you said that it did not work?)

Reply:
Yes. It did not work. I got the same linker errors that were resolved with all of this nonsense when running the native config.